lib/gems/pending/metadata/MIQExtract/MIQExtract.rb

```diff
@@ -158,10 +158,10 @@ def extract(category, &blk)
   end
 
   def getVMConfig(_c)
-    # Get VM config in XML format
-    config_xml = @target.vmConfig.toXML(true, @target)
-
     begin
+      # Get VM config in XML format
+      config_xml = @target.vmConfig.toXML(true, @target)
```

Comment:
I'm not sure this is the right solution to the problem.

Moving this call into the begin block will cause any failures to be silent.
This call shouldn't fail. The root cause of the problem is in the `MiqOpenStackImage.rb` and `MiqOpenStackInstance.rb` files. The corresponding objects delegate calls to a `MiqVm` object. The problem is `vmConfig` is not in the `SUPPORTED_METHODS` array, so the code doesn't know to delegate that method.

The proper solution may be just to add `:vmConfig` to the `SUPPORTED_METHODS` arrays.